frontend/src/components/common/form/ParentSelect.tsx

```diff
@@ -165,15 +178,16 @@ export const ParentSelect: React.FC<IParentSelect> = ({
                   if (i.parent) {
                     return (
                       <MenuItem key={index + 1} option={i} position={index + 1}>
-                        <Highlighter search="">{i.label}</Highlighter>
+                        <Highlighter search="">
+                          {i.parentId === -1 ? <b>{i.label}</b> : i.label}
```

Comment:
Bold the childless parent agencies so they appear like the other parent agencies